src/main/java/nimbus/command/FindCommand.java

```diff
@@ -0,0 +1,56 @@
+package nimbus.command;
+
+import nimbus.exception.WrongDateTimeFormatException;
+import nimbus.task.Task;
+import nimbus.ui.TaskList;
+import nimbus.ui.Ui;
+
+import java.util.ArrayList;
+
+/**
+ * This commands finds the task by searching for a keyword in the task description
+ */
+
+public class FindCommand extends Command {
+    private String userInput;
+    private ArrayList<Task> tasks;
+
+    /**
+     * Creates a FindCommand object containing the keyword as user input
+     * and taskList to search for
+     *
+     * @param userInput the keyword the user provides
+     * @param taskList task list to search in
+     */
+
+    public FindCommand(String userInput, TaskList taskList) {
+        super(userInput, taskList);
+        this.userInput = userInput;
+        this.tasks = taskList.getTaskList();
+    }
+
+    /**
+     * Iterates through the arraylist to find tasks with the keyword
+     * Prints out the tasks with the keyword
+     */
+
+    @Override
+    public void execute() {
+        if (userInput.length() < 6) {
+            System.out.println("Please enter what you want to find" +
```

Comment:
Should this operator (+) be on the next line? I think the problem is around for some other places too.